deepchem/models/torch_models/readout.py

```diff
@@ -0,0 +1,85 @@
+from typing import List
+try:
+    import torch
+    import torch.nn as nn
+except ModuleNotFoundError:
+    raise ImportError('The module requires PyTorch to be installed')
+
+from deepchem.models.torch_models.attention import SelfAttention
+
+
+class GroverReadout(nn.Module):
+    """Performs readout on a batch of graph
+
+    The readout module is used for performing readouts on batched graphs to
```

Comment:
Can you add a few sentences explaining how this is used in the Grover model?

Reply:
added docs in 521a676